packages/shared/src/supabase-browser.ts

```diff
@@ -0,0 +1,72 @@
+import { createClient, Session } from '@supabase/supabase-js';
+import { parse, serialize } from 'cookie';
+import { CookieOptions } from './types';
+import { isBrowser } from './utils/helpers';
+
+export function createBrowserSupabaseClient({
```

Comment:
With the new v2 createClient API shouldn't we be expecting a users to provide a Database type?